phpunit.xml.dist

```diff
@@ -2,10 +2,10 @@
 	bootstrap="tests/php/bootstrap.php"
 	backupGlobals="false"
 	colors="true"
-	convertDeprecationsToExceptions="true"
-	convertErrorsToExceptions="true"
-	convertNoticesToExceptions="true"
-	convertWarningsToExceptions="true"
```

Comment:
This can be reverted again once there are no warnings from WP core anymore (looking at you, Requests library)